components/LinkCardComponent.tsx

```diff
@@ -0,0 +1,79 @@
+import React, { useState } from "react";
+import styles from "../styles/FolderPage.module.css";
+import { formatDate, setBeforeDate } from "../utils/commonUtil";
+import { styled } from "styled-components";
+import CardDropdownComponent from "./CardDropdownComponent";
+import Link from "next/link";
+
+type props = {
+  imgSrc?: string;
+  createdAt: string;
+  desc?: string;
+  url: string;
+  type?: "default" | "static";
+};
+
+const LinkCardComponent = ({
+  imgSrc,
+  createdAt,
+  desc,
+  url,
+  type = "default",
+}: props) => {
+  const [show, setShow] = useState(false);
+
+  const curDate = new Date(createdAt);
+  const noImage = "/images/card_no_img.png";
+
+  return (
+    <Link href={url} target="_blank" className={styles.card_link}>
```

Comment:
target은 사용해주지 않아도 괜찮을 것 같네요!
```suggestion
    <Link href={url} className={styles.card_link}>
```